src/VirtoCommerce.Platform.Web/Views/Home/Index.cshtml

```diff
@@ -46,5 +88,63 @@
     <div class="error-detail" ng-if="platformError.detailVisible" ng-bind-html="platformError.detail"></div>
 </div>
 
+
+<div class="error-feature" style="display:none">
+    <h1 style="padding-top: 50px; padding-left: 50px; ">Startup error</h1>
+    <p style="padding: 15px 0 10px 50px; font-size: 24px">The script injection error occurred:</p>
+    <textarea  id="scriptError" class="textarea-error error-text" readonly></textarea>
+    <div class="error-content" style="display:none">
+        <p style="padding-top: 10px; padding-left: 50px; font-size: 24px;">The following modules were loaded with errors:</p>
+        <div id="moduleInfo" class="error-text">
+        </div>
+    </div>
+</div>
 <ui-view></ui-view>
 
+
+<style>
```

Comment:
can we move these styles to https://github.com/VirtoCommerce/vc-platform/tree/master/src/VirtoCommerce.Platform.Web/wwwroot/css/themes/main/sass/modules